### PR TITLE
DDT-29: event-subscription filters

### PR DESCRIPTION
**Pull request by Abdiiir**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	<groupId>org.dcsa</groupId>
 	<artifactId>dcsa_tnt</artifactId>
 	<version>0.0.1-SNAPSHOT</version>
-	<packaging>jar</packaging>
+	<packaging>war</packaging>
```
Comment (douradofrazer) [inline pom.xml:14]: We want to keep the `jar` packaging. We're not using `war`.

File: pom.xml [hunk 2]
```diff
@@ -25,17 +25,92 @@
 		<dcsa.artifacttype>-SNAPSHOT</dcsa.artifacttype>
 	</properties>
 
+
 	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-r2dbc</artifactId>
```
Comment (douradofrazer) [inline pom.xml:32]: `dcsa_core` and  `dcsa_event_core` already include all these dependencies, so you do not need to add them here.

File: src/main/java/org/dcsa/tnt/Application.java [hunk 3]
```diff
@@ -1,11 +1,17 @@
...
 
+import org.dcsa.core.repository.ExtendedRepositoryImpl;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.data.r2dbc.repository.config.EnableR2dbcRepositories;
 
 @SpringBootApplication
+@ComponentScan("org.dcsa")
+@EnableR2dbcRepositories(basePackages = {"org.dcsa"}, repositoryBaseClass = ExtendedRepositoryImpl.class)
```
Comment (douradofrazer) [inline src/main/java/org/dcsa/tnt/Application.java:11]: These configurations have been moved to `ApplicationConfig`, not required here anymore.

File: src/main/java/org/dcsa/tnt/ServletInitializer.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
+package org.dcsa.tnt;
+
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
+
+public class ServletInitializer extends SpringBootServletInitializer {
```
Comment (douradofrazer) [inline src/main/java/org/dcsa/tnt/ServletInitializer.java:6]: We do not require this file as we are not using tomcat.

File: src/main/java/org/dcsa/tnt/controller/EventController.java [hunk 5]
```diff
@@ -31,9 +26,4 @@ public EventService getService() {
     protected ExtendedRequest<Event> newExtendedRequest() {
         return new ExtendedGenericEventRequest(extendedParameters, r2dbcDialect);
     }
-
-    @Override
-    public Mono<Event> create(@Valid @RequestBody Event event) {
```
Comment (douradofrazer) [inline src/main/java/org/dcsa/tnt/controller/EventController.java:36]: This is required as we are to disable the endpoints as per recent requirements.

File: src/test/java/org/dcsa/tnt/controller/EventControllerTest.java [hunk 6]
```diff
@@ -1,99 +0,0 @@
...
-import org.springframework.security.web.server.SecurityWebFilterChain;
-import org.springframework.test.web.reactive.server.WebTestClient;
-import org.springframework.web.reactive.function.BodyInserters;
-
-import java.time.OffsetDateTime;
-import java.util.UUID;
-
-@DisplayName("Tests for Event Controller")
-@WebFluxTest(controllers = EventController.class)
-class EventControllerTest {
```
Comment (douradofrazer) [inline src/test/java/org/dcsa/tnt/controller/EventControllerTest.java:29]: Why is the test file being removed?

**Comment by Abdiiir** [comment]
The commented changes are from other commits. I will push a branch. 

You can ignore this pull request+

**Pull request closed without merging**